mindmeld/active_learning/classifiers.py

```diff
@@ -183,6 +248,13 @@ def _get_probs(
         Returns:
             prob_vector (List[List]]): Probability distribution vectors for given queries.
         """
+        if nlp_component_type == "entity":
+            return MindMeldALClassifier._get_entity_probs(
+                classifier=classifier,
+                queries=queries,
+                tag_to_id=nlp_component_to_id,
+            )
+
         queries_prob_vectors = []
```

Comment:
Consider placing the code below in a separate helper function as well "_get_classifier_probs()"